vizro-core/src/vizro/models/_components/ag_grid.py

```diff
@@ -114,7 +107,9 @@ def build(self):
         return dcc.Loading(
             [
                 html.H3(self.title, className="table-title") if self.title else None,
-                html.Div(grid, id=self.id, className="table-container"),
+                # The pagination setting (and potentially others) only work when the initially built AgGrid has the same
+                # setting as the object that is built on-page-load and rendered finally.
```

Comment:
Not related to this change so feel free to ignore here, but just spotted a weird looking inconsistency that I guess is probably correct but worth checking...

The structure here is:
```
dcc.Loading(
[
    html.H3(), html.Div(id=self.id, className="table-container")
])
```

while for `Table` it's this:
```
dcc.Loading(
    html.Div(
        [
            html.H3(), html.Div(id=self.id),
         ],
    className="table-container",
))
```

So there seems to be an extra `Div` in the `Table` case the the `className="table-container"` doesn't apply to the table itself.

Is that correct how it is?

Reply:
You can definitely try removing it from the Table as well. 

However, I remember there were some overflowing issues when the Table was used inside the Container inside the Tab, but I am not sure if it's related to this outer container. So if you remove it and it still looks fine and passes all of Alexeys screenshot tests, I think it's all good 👍

Reply:
Let's do this in another PR - I think this may drag things out, and is off-topic for this one. But will uncomment on merge